docs/detections/rules-ui-manage.asciidoc

```diff
@@ -101,7 +104,8 @@ NOTE: You cannot export Elastic prebuilt rules.
 .. Drag and drop the file that contains the detection rules.
 +
 NOTE: Imported rules must be in an `.ndjson` file.
-.. Select *Overwrite existing detection rules with conflicting Rule ID* to update existing rules if they match any rules in the import file. Configuration data included with the rules, such as actions, are also overwritten.
+.. Select *Overwrite existing detection rules with conflicting "rule_id"* to update existing rules if they match the `rule_id` value of any rules in the import file. Configuration data included with the rules, such as actions, are also overwritten.
+.. Select *Overwrite existing exception lists with conflicting "list_id"* to update existing exception lists associated with rules if they match the `list_id` value of any rules in the import file.
```

Comment:
I think saying that they are "updated" if overwrite is selected is not totally accurate here. What we do behind the scenes if we find a list with a matching `list_id` is delete the existing one and it's items, then create the list using the imported one.

Reply:
Thanks @yctercero for the context. How about this:
```suggestion
.. Select *Overwrite existing exception lists with conflicting "list_id"* to replace existing exception lists with exception lists from the import file if they match the `list_id` value.
```

Reply:
Sounds good!